docs/test/anchors.test.js

```diff
@@ -0,0 +1,23 @@
+var fs = require('fs');
+const cheerio = require('cheerio');
+
+const $ = cheerio.load(fs.readFileSync('_site/index.html'));
```

Comment:
```suggestion
const $ = cheerio.load(fs.readFileSync(path.join('_site', 'index.html')));
```

Reply:
Not necessarily needed to work, but getting in the habit of using Windows-friendly paths can save some headaches